schemas/event/1.0/schema.json

```diff
@@ -42,11 +44,13 @@
         },
         "startDate": {
             "type": "string",
-            "format": "date"
+            "format": "date",
+            "pattern": "^\\d{4}-\\d{2}-\\d{2}$"
```

Comment:
Since format is already set to date, is setting pattern required ? Is the JSON schema validation not working out of the box ?

Reply:
if we set only format then allow minus values in date format that why we add this
@coolbung

Reply:
@maheshkumargangula Please review code